src/attributecode/transform.py

```diff
@@ -129,77 +129,24 @@ def transform_json(data, transformer):
     renamings = transformer.column_renamings
     #if json is output of scancode-toolkit
     if(data["headers"][0]["tool_name"] == "scancode-toolkit"):
-        new_data, errors = process_json_keys_scancode(data, renamings, transformer)
-
-    elif isinstance(data, list):
-            for item in data:
-                element, err = process_json_keys(item, renamings, transformer)
-                for e in element:
-                    new_data.append(e)
-                for e in err:
-                    errors.append(e)
+        #only takes data inside "files"
+        data = data["files"]
+        #automatically renames path to about_resource
+        if("path" not in renamings.keys()):
+            renamings["path"] = "about_resource"
```

Comment:
@srthkdb I tend not to do this automatically in code side as, who knows, scancode may change the variable name in the future (thou it's very less likely), or AboutCode TK may change the "about_resource" to "path".

Reply:
Removed it!